sycl/test/usm/allocator_vector.cpp

```diff
@@ -1,4 +1,3 @@
-// XFAIL: cuda
 // piextUSM*Alloc functions for CUDA are not behaving as described in
```

Comment:
This test is failing in our CI.
http://ci.llvm.intel.com:8010/#/builders/37/builds/705/steps/15/logs/FAIL__SYCL__allocator_vector_cpp

Any ideas why?

Reply:
I can have a look later today or tomorrow.

Reply:
+@bjoernknafla, just in case he might know if this is a known issue.

Reply:
That error could come from CUDA event handling needing a thread pool as the CUDA callbacks currently used cannot issue CUDA API calls - and in some cases the callback holds the last existing reference to an event.

If I understand correctly, the the following PR is meant as a way towards a solution: https://github.com/intel/llvm/pull/1471